draft-packages/empty-state/KaizenDraft/EmptyState/README.mdx

```diff
@@ -40,23 +40,25 @@ import Dont from "docs-components/Dont"
 
 * There are different Empty States including:
```

Comment:
We have options for border style listed at the top of the page but I've never seen us mix and match border style with the empty states and I don't think we should start doing that. Can we clear that information out?

Reply:
Yeah we're not even using them in the Heart states which have become a lot more concrete since then – I'll remove it.